.github/workflows/format-code.yml

```diff
@@ -10,6 +10,22 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
     steps:
+      - name: 'Post message to show that the command has been triggered'
+        id: status-comment
+        uses: actions/github-script@47f7cf65b5ced0830a325f705cad64f2f58dddf7 # 3.1.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          result-encoding: string
+          script: |
+            const comment = await github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Formatting...'
+            })
+
+            return comment.id
+
```

Comment:
Scratch this all, this should really be a check run 🤦

Reply:
Okay, let me know when to review!